lib/stackInformation.js

```diff
@@ -181,4 +181,15 @@ module.exports = {
 		});
 	},
 
+	aliasGetAliasLatestFunctionVersionByFunctionName(aliasName, functionName) {
+		return this._provider.request('Lambda',
+			'getAlias',
+			{ FunctionName: functionName, Name: aliasName },
+			this._options.stage,
+			this._options.region)
+			.then(result => {
+				return result.FunctionVersion
```

Comment:
You can use the short form here:
```
.then(result => _.get(result, 'FunctionVersion', null));
```
With lodash that will also cover all cases where result might be invalid and returns a reasonable default.
Of course the result should then be checked at the caller (above).